app/routes/docs/history.mdx

```diff
@@ -0,0 +1,48 @@
+---
+handle:
+  breadcrumb: History
+---
+
+import history from '~/img/gcn_history.png'
+
+# History of GCN
+
+<img
+  src={history}
+  width="900"
+  height="250"
```

Comment:
This doesn't match the dimensions of the image file.

Reply:
updated with linearly scaled image